modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -49,6 +49,13 @@ menuconfig BUILD_WITH_TFM
 
 if BUILD_WITH_TFM
 
+config TFM_SECURE_UART1
+	bool "Use UART1 for TF-M output"
+	default y
+	help
+	  Reserve the UART1 port for TF-M output. This makes the UART1
+	  unavailable to the non-secure application.
+
```

Comment:
Does this option also affect whether UART1 is configured as secure or non-secure?

Reply:
Yes, when this is disabled the peripheral will be configure as non-secure:
https://github.com/nrfconnect/sdk-trusted-firmware-m/blob/e0ea69fe589aaf24ddaf49e8e0d584abae39b159/trusted-firmware-m/platform/ext/target/nordic_nrf/common/nrf9160/target_cfg.c#L198